xml/System.Windows.Forms/SplitterPanel.xml

```diff
@@ -285,8 +250,8 @@
         <ReturnType>System.Windows.Forms.Padding</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the space, in pixels, that is specified by default between controls.</summary>
```

Comment:
how about: `Gets the default space, in pixels, that separates the controls .`

Reply:
```suggestion
        <summary>Gets the default space, in pixels, that separates the controls.</summary>
```

Reply:
@mairaw feel free to merge the suggestion if you think it improves the summary.